client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -182,6 +182,13 @@ const TabDetailsContent: React.FC<{
         .filter((fullArchetype) => fullArchetype?.review)
         .filter(Boolean);
 
+  const { task, isFetching, fetchError, refetch } = useFetchTaskByID(
```

Comment:
```suggestion
  const { task } = useFetchTaskByID(
```

No need to define the variables that are not used.

Reply:
I implemented your suggestion. Thank you.